src/Raven.Client/Raven.Client.csproj

```diff
@@ -1,6 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Version></Version>
+    <Version>
+    </Version>
```

Comment:
Are you sure splitting this into 2 lines won't break some regex value update here?

Reply:
```
$ grep '<Version>' -R  scripts/*
grep: scripts/assets/bin/nuget.exe: binary file matches
scripts/updateSourceWithBuildInfo.ps1:    $versionPattern = [regex]'(?sm)<Version>[A-Za-z0-9-\.\r\n\s]*</Version>'
scripts/updateSourceWithBuildInfo.ps1:    $result = $versionPattern.Replace($inputText, "<Version>$version</Version>")
```

:)

Reply:
Fixed